src/data-tutorials/users.tsx

```diff
@@ -938,6 +938,14 @@ const Users: User[] = [
     website: '/blog/tutorials/dynamic-nft-creation-using-svg-to-build-in-smart-contract-on-celo',
     tags: ['intermediate', 'solidity','celosage','erc721','truffle'],
   },
+  {
+    title: 'Deploy and Interact with Smart Contracts on the Celo Blockchain',
```

Comment:
You already added this block down but now added this now 2 times

Reply:
@developerkunal 

I removed one already. Thanks